### PR TITLE
Database separation and server structure

### PR DESCRIPTION
**Pull request by DylanBulmer**
## Database modifications
I dissected the massive SQL file into six smaller files in accordance to our software architecture. In addition to this I structured the files into their own folders in respect to theirpurpose, ie `./db/<service>/Skoot-<service>.sql` and created Dockerfiles for each database to compile into a container. 

## Server structure modifications
To start moving towards a microservice architecture, I moved our current work into the gateway service, also moving the account route into the account service. Finally I initialize all the services. 

Before completing the merge, I need to complete the following:
- [x] Add a `Dockerfile` to each service *(excluding the databases)*
- [x] Finalize the gateway API code structure.
- [x] Finalize the `docker-compose.yml` file 

## Other notes
Please let me know if more should be done before diving into individual services.

### CODE REVIEW

**Comment by DylanBulmer** [comment]
Finished setting up docker, now starting work on structuring the Gateway API to accept SocketIO requests and forward then to the correct service API.

**Comment by DylanBulmer** [comment]
Everything is working besides `docker-compose up`. Not the creation and running of the mircoservices, but the rebuilding of them is not working properly.

**Comment by DylanBulmer** [comment]
Docker has been fixed.

File: skoot-server/db/account/Skoot-Account.sql [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `skoot`.`user_setting`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `skoot`.`user_setting` (
+  `user_id` INT NOT NULL AUTO_INCREMENT,
+  `gradiant_start` VARCHAR(6) NULL DEFAULT NULL,
+  `gradian_end` VARCHAR(6) NULL DEFAULT NULL,
```
Comment (NathanRGagnon) [inline skoot-server/db/account/Skoot-Account.sql:58]: gradian? Not sure if this will matter but we should probably change it to gradient.

Comment (EngineerFaunce) [reply skoot-server/db/account/Skoot-Account.sql:58]: @NathanRGagnon Good catch

Comment (DylanBulmer) [reply skoot-server/db/account/Skoot-Account.sql:58]: gradiant_start too...

Comment (DylanBulmer) [reply skoot-server/db/account/Skoot-Account.sql]: Fixed these issues

**Pull request merged**